src/signals/incident-management/containers/StandardTextsAdmin/components/DetailPage/utils.ts

```diff
@@ -2,6 +2,12 @@
 // Copyright (C) 2023 Gemeente Amsterdam
 import type { StandardTextDetailData, StandardTextForm } from './types'
 
+export const isNumeric = (str?: string) => {
```

Comment:
Why would you need this util? You can just check if `id` is truthy or undefined in the code?

Reply:
Have since removed this function.